### ISSUE TITLE
Cocoapod support

### ISSUE THREAD

**Issue by djones6** [swift-server/async-http-client#114]
In order for other Cocoapod-enabled packages to adopt this one (such as [SwiftyRequest](https://github.com/IBM-Swift/SwiftyRequest/pull/76)), an async-http-client pod is needed.

Swift-NIO et. al is already available in pod form, so I believe this should be relatively straightforward.

**Comment by weissi** [comment]
agreed, should be super simple with the same (slightly modified) script NIO uses

**Comment by weissi** [comment]
We should do this as soon as 1.0.0 has been released.
